src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -93,6 +93,133 @@ function TimePeriodSelector(props) {
   );
 }
 
+const behaviouralResponseReforms = {
+  uk: {
+    "gov.simulation.capital_gains_responses.elasticity": {
+      "2024-01-01.2100-01-01": -0.7,
+    },
+    "gov.simulation.labor_supply_responses.income_elasticity": {
+      "2024-01-01.2100-01-01": -0.05,
+    },
+    "gov.simulation.labor_supply_responses.substitution_elasticity": {
+      "2024-01-01.2100-01-01": 0.25,
+    },
+  },
+  us: {},
+};
+
+function BehaviouralResponseToggle(props) {
+  // Selector like the full lite toggle that toggles between 'static' and 'dynamic' versions of the dataset.
+  // should set a query param with mode=static or mode=dynamic
+
+  const { policy, metadata } = props;
+  const [searchParams, setSearchParams] = useSearchParams();
+  const value = searchParams.get("mode") || "static";
```

Comment:
I think "mode" as an option might be pretty nebulous here. It may be clearer to do something like "dynamic=true", with the default behavior being "dynamic=false", which would signal a static run.